include/reshade_api.hpp

```diff
@@ -743,5 +755,15 @@ namespace reshade { namespace api
 		/// <param name="name">Name of the definition.</param>
 		/// <param name="value">Value of the definition.</param>
 		virtual void set_preprocessor_definition_for_effect(const char *effect_name, const char *name, const char *value) = 0;
+
+		/// <summary>
+		/// Activates ReShade's overlay programatically.
+		/// </summary>
+		/// <param name="activate">Activation state to request.</param>
+		/// <param name="source">Source of activation request.</param>
+		/// <remarks>
+		/// Returns <see langword="true"/> if the activation was accepted, otherwise returns <see langword="false"/>.
+		/// </remarks>
+		virtual bool activate_overlay(bool activate, reshade::api::input_source source) = 0;
```

Comment:
Similarily to the event, could then rename this `open_overlay()`. Makes it consistent with other events like `reshade_set_technique_state` or `reshade_reorder_techniques`, where the `effect_runtime` method name is the same as the event (apart from the added `reshade_` prefix) that is invoked when the action it triggers occurs.